proxy/proxy_handlers.go

```diff
@@ -0,0 +1,87 @@
+package proxy
+
+import (
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/websocket"
+	"github.com/taskcluster/webhooktunnel/wsmux"
+)
+
+func (p *proxy) handler(w http.ResponseWriter, r *http.Request) {
+	// register a worker
+	log.Printf("new request: %s", r.URL.Path)
+	if strings.HasPrefix(r.URL.Path, "/register") {
```

Comment:
r.URL.Path == "/register" will not work since the worker ID is sent in the url. I'm guessing the header field should not contain the worker ID but rather the json auth token.